src/Microsoft.Azure.SignalR.Management/TaskExtensions.cs

```diff
@@ -28,5 +30,20 @@ public static async Task OrTimeout(this Task task, CancellationToken cancellatio
                 }
             }
         }
+
+        public static async Task<RestApiEndpoint> WithTracingIdAsync(this Task<RestApiEndpoint> task, bool enable = false)
```

Comment:
why we need `enable` parameter? isn't `WithTracingIdAsync` meaning enable?

Reply:
going to put the `enable` to `RestClient.SendAsync`